insights/specs/default.py

```diff
@@ -606,6 +606,7 @@ def sap_sid(broker):
     sap_hdb_version = foreach_execute(sap_sid, "/usr/bin/sudo -iu %sadm HDB version", keep_rc=True)
     saphostctl_getcimobject_sapinstance = simple_command("/usr/sap/hostctrl/exe/saphostctrl -function GetCIMObject -enuminstances SAPInstance")
     sat5_insights_properties = simple_file("/etc/redhat-access/redhat-access-insights.properties")
+    satellite_content_hosts_count = simple_command("/usr/bin/su - postgres -c \"psql -d foreman -c 'select count(*) from hosts'\"")
```

Comment:
@huali027 this spec should use the `sudo` command instead:
```
"/usr/bin/sudo -iu postgres psql -d foreman -c 'select count(*) from hosts'"
```

Reply:
@bfahr, sure, I'll update it, thanks for your review.